### ISSUE TITLE
Clear volatile PNOR sections on power offs

### ISSUE THREAD

**Issue by geissonator** [openbmc/openbmc#2397]
<cde:info> SW407241 </cde:info>
PNOR sections have a "volatile" flag which indicate it should be cleared outside of the automatic reboot path.  This story will be used to provide a function to clear these out and to call it in the appropriate place

https://github.com/open-power/pnor/blob/master/p9Layouts/defaultPnorLayout_128.xml
```
    <volatile/>     -> Indicates Partition is not preserved across power offs, 
                       but is across reboots. BMC will clear on power off/on
```

**Comment by anoo1** [comment]
- [ ] Get partition with volatile flag and add this new flag to the pnor table of contents.
- [ ] Create function to get volatile partition and clear it.
- [ ] Call function during power off.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/9544> Service files to clear volatile PNOR partitions on host reboot
Resolves  #2397 Clear volatile PNOR sections on power offs
